### ISSUE TITLE
How to allow browser's address bar camera and microphone permissions 

### ISSUE THREAD

**Issue by karishmaibentos** [gree/unity-webview#940]
I m facing issue in using webview for android in unity . Issue
Your browser needs camera and microphone access
1.Tap the lock icon in your browser's address bar
2.Tap "permissions"
3.Tap "camera and microphone", then tap allow both

Please reply asap.Thanks in advance.

**Comment by KojiNakamaru** [comment]
This plugin has no address bar. Do you really talk about this plugin?

**Comment by karishmaibentos** [comment]
Yes i am using this plugin for video audio chat in unity android webview but facing the issue ... screen shows this prompt-Your browser needs camera and microphone access. Can we do video audio chat in this webview android plugin?

**Comment by KojiNakamaru** [comment]
I've put a modified sample that can be directly opened with 2020.3.48f1.

https://github.com/gree/unity-webview/tree/example-for-940/sample

You can see required modifications (described at https://github.com/gree/unity-webview/tree/08e8ec94ce4756b049f129b76195595f445849d8#cameraaudio-permissionfeature) for camera/microphone in the following commits.

https://github.com/gree/unity-webview/compare/60adc71...89f94e8

In short,

* define `UNITYWEBVIEW_ANDROID_ENABLE_CAMERA` and `UNITYWEBVIEW_ANDROID_ENABLE_MICROPHONE`
* call `webViewObject.SetCameraAccess(true)` and `webViewObject.SetMicrophoneAccess(true)`
* request required permissions
